torch/distributed/_tensor/api.py

```diff
@@ -301,19 +317,33 @@ def from_local(
             local_tensor, device_mesh, tuple(placements), run_check
         )
 
-    def to_local(self) -> torch.Tensor:
+    def to_local(
+        self, *, grad_placements: Optional[Sequence[Placement]] = None
+    ) -> torch.Tensor:
         """
         Get the local tensor of this DTensor on its current rank. For sharding it returns
         a local shard of the logical tensor view, for replication it returns the replica on
         its current rank.
 
+        Keyword args:
+            grad_placements (List[:class:`Placement`], optional): the placements that
+                describes the gradient layout of the local torch.Tensor on DeviceMesh.
```

Comment:
Maybe a clearer way to describe this, if I understand correctly: the placement describes the future layout of any gradient of the tensor returned from this function.

Reply:
sounds great, thanks updated!